server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -143,6 +143,27 @@ export class RwcdsFormService {
       &$expand=dcp_ZoningResolution
     `);
 
+    // dcp_includezoningtextamendment is a field on the rwcds-form that should be "Yes" (717170000) if...
+    // there exists a "ZR" action and that action's dcp_ZoningResolution.dcp_zoningresolution is "AppendixF"
+    const zrAppendixF = projectActions.filter(action =>
+      action['_dcp_action_value@OData.Community.Display.V1.FormattedValue'] === 'ZR'
+      && action.dcp_ZoningResolution.dcp_zoningresolution === 'AppendixF'
+    );
+
+    const includeZoningTextAmendment = zrAppendixF.length > 0;
+
+    if (includeZoningTextAmendment) {
+      await this.crmService.update('dcp_rwcdsforms', dcp_rwcdsformid, {
+        // set to "Yes"
+        dcp_includezoningtextamendment: 717170000,
```

Comment:
I know it seems tedious, but can we create an optionset for this here?
Like how we have the `PACKAGE_TYPE_OPTIONSET` in package.server.ts?

Reply:
For sure!